### ISSUE TITLE
[Bug]: LC0092 false positive for Action Areas - this is invalid as Areas are pre-defined by Microsoft

### ISSUE THREAD

**Issue by AlanDixey** [ALCops/Analyzers#537]
### Analyzer

LinterCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

LC0092

### Description

Custom Naming Patterns interfere with application of the diagnostic.  If a custom NamingPattern is set against Actions, it is applied to Action Areas as well, which is invalid as areas are pre-defined by Microsoft.

### AL code to reproduce

```al

```
actions
    {
        area(Processing)
        {
		action(Test)
		{}
	}
}

### Additional context

alcops.json to reproduce
{
    "NamingPatterns": {
        "Action": {
            "AllowPattern": "act[A-Za-z0-9]",
            "AllowDescription": "Should begin with 'act'."
        }
    }
}
